portfolio/src/main/java/Server2.java

```diff
@@ -0,0 +1,16 @@
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;/** Servlet that responds with a basic header text. */
+
+@WebServlet("/head")
+public class Server2 extends HttpServlet {
```

Comment:
nit: this is a servlet not a server, also server2 doesn't describe what it's doing!
    Suggestion: HelloWorldWalkthroughServlet